<launch>
    <arg name="robot_model" default="bez2" />
    <arg name="play_rosbag" default="false" />
    <arg name="record_rosbag" default="true" />
    <arg name="simulation" default="false" />

    <let name="motor_types" value="$(find-pkg-share soccer_firmware_interface)/config/motor_types.yaml" />
    <let name="motor_mapping" value="$(find-pkg-share soccer_firmware_interface)/config/$(var robot_model).yaml" />

    <node pkg="soccer_firmware_interface" name="soccer_firmware_interface"  exec="soccer_firmware" output="screen" emulate_tty="true">
        <param name="robot_model" value="$(var robot_model)" />
<!--         <param name="/use_sim_time" value="$(arg simulation)" /> -->
        <param name="motor_types" value="$(var motor_types)" />
        <param name ="motor_mapping" value="$(var motor_mapping)" />
    </node>


    </launch>
